Build failing in 2.7 and 3.4

```
======================================================================

ERROR: test_corr (skbio.stats.ordination.tests.test_util.TestUtils)

----------------------------------------------------------------------

Traceback (most recent call last):

  File "/home/travis/miniconda3/envs/env_name/lib/python3.4/site-packages/skbio/stats/ordination/tests/test_util.py", line 45, in test_corr

    obs = corr(self.small_mat)

  File "/home/travis/miniconda3/envs/env_name/lib/python3.4/site-packages/skbio/stats/ordination/_utils.py", line 173, in corr

    x = scale(x)

  File "/home/travis/miniconda3/envs/env_name/lib/python3.4/site-packages/skbio/stats/ordination/_utils.py", line 123, in scale

    a -= avg

TypeError: Cannot cast ufunc subtract output from dtype('float64') to dtype('int64') with casting rule 'same_kind'

----------------------------------------------------------------------
```

Looks like a package (numpy probably) has just been updated.

If I correct the dtype mismatch we get some dramatically different results:
```
======================================================================
FAIL: test_corr (__main__.TestUtils)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "skbio/stats/ordination/tests/test_util.py", line 49, in test_corr
    obs)
  File "/home/evan/.conda/envs/skbio/lib/python3.4/site-packages/numpy/testing/utils.py", line 494, in assert_almost_equal
    return assert_array_almost_equal(actual, desired, decimal, err_msg)
  File "/home/evan/.conda/envs/skbio/lib/python3.4/site-packages/numpy/testing/utils.py", line 886, in assert_array_almost_equal
    precision=decimal)
  File "/home/evan/.conda/envs/skbio/lib/python3.4/site-packages/numpy/testing/utils.py", line 708, in assert_array_compare
    raise AssertionError(msg)
AssertionError: 
Arrays are not almost equal to 7 decimals

(mismatch 100.0%)
 x: array([[ 0.3333333,  0.       , -0.3333333],
       [ 0.       ,  0.       ,  0.       ],
       [-0.3333333,  0.       ,  0.3333333]])
 y: array([[ 1.       ,  1.       , -0.9449112],
       [ 1.       ,  1.       , -0.9449112],
       [-0.9449112, -0.9449112,  1.       ]])
```

Any ideas?

--- a/skbio/stats/ordination/_utils.py
+++ b/skbio/stats/ordination/_utils.py
@@ -117,6 +117,7 @@ def scale(a, weights=None, with_mean=True, with_std=True, ddof=0, copy=True):
     """
     if copy:
         a = a.copy()
+    a = np.asarray(a, dtype=np.float64)
     avg, std = mean_and_std(a, axis=0, weights=weights, with_mean=with_mean,
                             with_std=with_std, ddof=ddof)
     if with_mean:

--- a/skbio/stats/ordination/tests/test_util.py
+++ b/skbio/stats/ordination/tests/test_util.py
@@ -43,9 +43,9 @@ class TestUtils(TestCase):
 
     def test_corr(self):
         obs = corr(self.small_mat)
-        npt.assert_almost_equal(np.array([[0.33333333, 0, -0.33333333],
-                                          [0, 0, 0],
-                                          [-0.33333333, 0, 0.33333333]]),
+        npt.assert_almost_equal(np.array([[1, 1, -0.94491118],
+                                          [1, 1, -0.94491118],
+                                          [-0.94491118, -0.94491118, 1]]),
                                 obs)
 
     def test_corr_shape_mismatch(self):

